### PR TITLE
updated credits in README

### PR DESCRIPTION
**Pull request by SarahCreager**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,74 @@
...
+      const API_URL = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.exploreQuery}&format=json`;
+      const response = await axios.get(API_URL);
+      console.log(response.data);
+      this.setState({location:response.data[0]});
+      this.setState({image:`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_CITY_KEY}&center=${this.state.location.lat},${this.state.location.lon}&zoom=15`});
+    } catch {
+      this.setState({error: true});
+    }
+  }
+
```
Comment (RivaD2) [inline src/App.js:36]: Nice work on incorporating try catch blocks! :)

When you can make sure to read a bit more on these and how they cause some issues in React. Really nice working this in 👍 

Note: Just remove those helpful logs when you have things a workin'/before you merge PR :)

File: src/App.js [hunk 2]
```diff
@@ -42,21 +42,21 @@ export default class App extends Component {
...
           {this.state.error ? <h2>not a valid location</h2>
             : this.state.location.place_id &&
           <>
-            <h2>City Name: {this.state.location.display_name}</h2> 
-            <h3>Latitude: {this.state.location.lat}.</h3> 
+            <h2>City Name: {this.state.location.display_name}</h2>
+            <h3>Latitude: {this.state.location.lat}.</h3>
             <h3>Longitude: {this.state.location.lon}. </h3>
             <Image src={this.state.image} alt="map" thumbnail></Image>
           </>}
```
Comment (Beers15) [inline src/App.js]: Hi Sarah this code looks good!

To refactor it a bit you could take the container component that contains the button and text input field form and separate logic into its own component.

Same idea for the container with id="map". 

Then by passing a prop with a function to modify those state values in App.js this component could be cleaned up.

File: src/App.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ export default class App extends Component {
       error:'',
     };
   }
- 
+
   handleButtonClick = async () => {
     try {
       const API_URL = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.exploreQuery}&format=json`;
```
Comment (Beers15) [inline src/App.js]: Possibly add some logic for clearing the error message/state if the user proceeds to enter another accepted value into the form.

Comment (Beers15) [reply src/App.js]: The entire function was supposed to be cropped for the comment above.

**Pull request merged**